plugins/catalog/src/setupTests.ts

```diff
@@ -15,4 +15,4 @@
  */
 
 import '@testing-library/jest-dom';
-require('jest-fetch-mock').enableMocks();
+import 'whatwg-fetch';
```

Comment:
`node-fetch` didn't seem to work too nicely with the mocking, switching to the `github` polyfill which basically just wraps `XMLHttpRequest` seems to work best.

Reply:
Polyfilling the client is necessary for the code itself to run in Node. 

The Node API of MSW intercepts requests on the underlying implementation layer (`http`/`https` modules) and issues explicit `XMLHttpRequest` override for DOM-like environments (like jsdom, which implements `window.fetch` polyfill through XHR polyfill).

In Node, the library should work with any request issuing module (`node-fetch`, `whatwg-fetch`, `request`, etc.).